lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1364,6 +1364,9 @@ public function TicketChange($ticketid, array $props)
         if ($ticket['queueid'] != $props['queueid'] && isset($props['queueid'])) {
             $notes[] = trans('Ticket has been moved from queue $a to queue $b.', $LMS->GetQueueName($ticket['queueid']), $LMS->GetQueueName($props['queueid']));
             $type = $type | RTMESSAGE_QUEUE_CHANGE;
+            if (ConfigHelper::getConfig('rt.change_ticket_queue_resets_ticket_state') && ticket['state'] != RT_VERIFIED && ticket['state'] != RT_RESOLVED) {
```

Comment:
Może:
`rt.ticket_queue_change_resets_ticket_state`
?
Poza tym to chyba **boolean**, a jeśli tak i domyślnie _false_ to proponuję użyć ConfigHelper::checkConfig().
No i ChangeLog oraz wiki.